src/Modules/Workspaces/Commands.Workspaces.Test/RemovePowerBIWorkspaceUserTests.cs

```diff
@@ -0,0 +1,97 @@
+﻿/*
+* Copyright (c) Microsoft Corporation. All rights reserved.
+* Licensed under the MIT License.
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Management.Automation;
+using Microsoft.PowerBI.Commands.Profile.Test;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Microsoft.PowerBI.Commands.Workspaces.Test
+{
+    [TestClass]
+    public class RemovePowerBIWorkspaceUserTests
+    {
+        [TestMethod]
+        [TestCategory("Interactive")]
+        [TestCategory("SkipWhenLiveUnitTesting")] // Ignore for Live Unit Testing
+        public void EndToEndRemovePowerBIWorkspaceUserAsAdmin()
```

Comment:
Probably better to have these test names use "OrganizationScope" and "IndividualScope" since they more closely align with the PowerShell usage and existing tests.

Reply:
Resolved. Updated the test names.